client/blocks/stats-navigation/style.scss

```diff
@@ -111,6 +119,14 @@
 						top: 11px;
 						right: 0;
 					}
+
+
+					&:has(+ .page-modules-settings) {
+						.section-nav__mobile-header > svg {
```

Comment:
Updated in [a91ccff](https://github.com/Automattic/wp-calypso/pull/103325/commits/a91ccff5729717ef04af54b8ff69d5cad4172916).